packet/flatcar-linux/kubernetes/bootkube.tf

```diff
@@ -6,7 +6,7 @@ module "bootkube" {
   # Cannot use cyclic dependencies on controllers or their DNS records
   api_servers          = [format("%s-private.%s", var.cluster_name, var.dns_zone)]
   api_servers_external = [format("%s.%s", var.cluster_name, var.dns_zone)]
-  etcd_servers         = aws_route53_record.etcds.*.fqdn
+  etcd_servers         = [for device in packet_device.controllers: format("%s-etcd%d.%s.", var.cluster_name, index(packet_device.controllers, device), var.dns_zone)]
```

Comment:
Note, that this breaks the implicit dependency of bootkube module on `aws_route53_record.etdcs` resource. I think explicit dependency should be added then.

Reply:
We discussed, that bootkube itself should wait long enough before the records become available, so it should be fine, so long user does not specify `-parallelism=1`. I think with this, it will break.